lib/hex.ml

```diff
@@ -73,6 +73,9 @@ let of_string ?ignore s =
     None -> of_string_fast s
   | Some ignore -> of_helper ~ignore (fun i -> s.[i]) (String.length s)
 
+let of_bytes ?ignore b =
+  of_string ?ignore (Bytes.unsafe_to_string b)
```

Comment:
I think this should be `Bytes.to_string` -- the unsafe may break in the future with safe-string